ipn/ipnlocal/local.go

```diff
@@ -5077,9 +5071,9 @@ func (b *LocalBackend) authReconfigLocked() {
 	}
 
 	var flags netmap.WGConfigFlags
-	if prefs.RouteAll() {
-		flags |= netmap.AllowSubnetRoutes
-	}
+	// NOTE: we want to always allow subnet routes for the sake of enabling services
+	flags |= netmap.AllowSubnetRoutes
```

Comment:
I think it would be better to change what `AllowSubnetRoutes` means rather than always set this flag. For example, the `disableSubnetsIfPAC` thing below wouldn't do the right thing in this case (although I'm not sure if that's still used by anyone).

Instead, `cidrIsSubnet` in `wgengine/wgcfg/nmcfg/nmcfg.go` could check `tsaddr.IsTailscaleIP()` to return `false`.